src/behaviors/loginPrompt.js

```diff
@@ -1,35 +1,42 @@
 export default {
-  /**
-   * Init login prompt observer
-   * Called by Polymer on attach
-   * @return {undefined}
-   */
-  attached() {
-    this._promptLoginOnEditable();
+  properties: {
+
+    loginPrompt: {
+      type: Boolean,
+      observer: '_promptLoginOnEditable'
+    }
+
   },
 
   /**
    * Observer to prompt for login if editable && !authed
    * @return {undefined}
    */
-  _promptLoginOnEditable() {
-    let simplaLogin = this.$['login'],
+  _promptLoginOnEditable(loginPrompt) {
+    let { _simplaObservers: observers } = this,
+        simplaLogin = this.$['login'],
         promptLogin = (editable) => {
           if (editable && !this._authenticated) {
-            simplaLogin.prompt()
-              .then(loggedIn => Simpla.editable(loggedIn));
+            simplaLogin.prompt().then(loggedIn => {
+              Simpla.editable(loggedIn)
+            });
           }
         },
         initialPrompt = () => {
+          let shouldPrompt = Simpla.getState('editable') && loginPrompt;
```

Comment:
I don't think `&& loginPrompt` is needed? `initialPrompt` is only called lower down in the `if (loginPrompt) { ... }` block, so `initialPrompt` doesn't need to check it again.